src/libs/Trie/index.js

```diff
@@ -0,0 +1,115 @@
+import _ from 'underscore';
+import TrieNode from './TrieNode';
+
+class Trie {
+    constructor() {
+        this.root = new TrieNode();
+    }
+
+    /**
+    * Add a word to the Trie
+    * @param {String} word
+    * @param {Object} [metaData] - attach additional data to the word
+    * @param {TrieNode} node
+    * @param {Boolean} [allowEmptyWords] - empty word doesn't have any char, you shouldn't pass a true value for it because we are disallowing adding an empty word
+    * @returns {void}
```

Comment:
Missing `@returns` implies `void`

```suggestion
```

Reply:
It makes a lint error when removing, should we disable it?
![@returns](https://user-images.githubusercontent.com/108357004/195131255-25605cdc-b225-41e1-8867-afdad1babb98.png)

Reply:
Oh, I think it's because this function sometimes returns `null` instead of just `return`. If you intend it to be void, replace `return null;` with `return;`

Reply:
Done.